PSModule/AbstractLauncherTaskCmdlet.cs

```diff
@@ -49,14 +50,15 @@ protected override void ProcessRecord()
             string launcherPath, aborterPath = string.Empty, converterPath, paramFileName = string.Empty, resultsFileName;
             try
             {
-                Dictionary<string, string> properties = new Dictionary<string, string>();
+                Dictionary<string, string> properties;
                 try
                 {
                     properties = GetTaskProperties();

```

Comment:
It is better to check if the return value `properties` is null as this variable is used without protections later.